webplugin/js/app/kommunicate.js

```
@@ -492,16 +445,10 @@ $applozic.extend(true, Kommunicate, {
     isRichTextMessage: function (metadata) {
         // contentType should be 300 for rich text message in metadata
         // contentType 300 is removed from rich message payload since Jan-2020 and old payload this may getting used.
-        return (
-            metadata &&
-            (metadata.hasOwnProperty('templateId') ||
-                metadata.contentType == 300)
-        );
+        return metadata && (metadata.hasOwnProperty('templateId') || metadata.contentType == 300);
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)